readme.rst

```diff
@@ -11,10 +11,13 @@
 
     Release branch: |last_release_build|
 
-.. |last_release_build| image:: https://ci.navitia.io/job/navitia_release/badge/icon
-    :target: https://ci.navitia.io/job/navitia_release/
+.. |last_release_build| image:: https://github.com/CanalTP/navitia/workflows/Build%20Navitia%20Packages%20For%20Release/badge.svg
```

Comment:
You can probably use shields to "flat-square" it too, like https://github.com/CanalTP/transit_model

Reply:
yes it is possible. I didn't think about that flat-square. Thanks